File: src/test/java/no/utgdev/diy/ex1/Ex1Config.java
package no.utgdev.diy.ex1;

//REMOVE import no.utgdev.diy.annotations.Bean;
//REMOVE import no.utgdev.diy.annotations.Import;

import java.util.Collections;
import java.util.List;

//REMOVE @Import(Ex1Config2.class)
public class Ex1Config {
    //REMOVE @Bean
    public List<String> list() {
        return Collections.emptyList();
    }

    //REMOVE @Bean
    public String string() {
        return "";
    }
}

File: src/test/kotlin/no/utgdev/diy/ex1/Ex1Config.kt
package no.utgdev.diy.ex1

//REMOVE import no.utgdev.diy.annotations.Bean
//REMOVE import no.utgdev.diy.annotations.Import

//REMOVE @Import([Ex1Config2::class])
class Ex1Config {
    //REMOVE @Bean
    fun list(): List<String> {
        return emptyList()
    }

    //REMOVE @Bean
    fun string(): String {
        return ""
    }
}